src/plone/restapi/tests/mixin_ordering.py

```diff
@@ -195,6 +195,38 @@ def test_reorder(self):
             self.folder.contentIds(),
         )
 
+    def test_ordering_preserves_bytestring_obj_id(self):
```

Comment:
`bytestring` -> `native_string`
(See https://portingguide.readthedocs.io/en/latest/strings.html#the-native-string)